test/renderer/shader_registry.test.cpp

```diff
@@ -206,6 +209,7 @@ TEST(ShaderRegistry, NamedReplace) {
     ASSERT_NE(progA, progB);
 }
 
+#ifdef MLN_RENDER_BACKEND_OPENGL
```

Comment:
It's already disabled.

Does it pass if you enable it now?

Reply:
Still disabled, probably not passing, I didn't fix them. Should compile only for GL only, the code is GL specific.